### PR TITLE
Add Organization, Area & Iteration

### PR DESCRIPTION
**Pull request by aslakknutsen**

### CODE REVIEW

File: _blueprints/model_overview.adoc [hunk 1]
```diff
@@ -3,25 +3,30 @@
...
+	USER -> TEAM_ROLE
+	LOGIN -> WORKITEM[label="CUD|Assigned"]
 	
 	TEAM_ROLE -> TEAM
 	TEAM_ROLE -> PERMISSION
 
 	TEAM -> PROJECT
-	
-	
+	ORGANIZATION -> TEAM
```
Comment (tsmaeder) [inline _blueprints/model_overview.adoc:15]: Are there not teams associated with projects? Contributors to "almighty-core", for example?

File: _blueprints/model_overview.adoc [hunk 2]
```diff
@@ -3,25 +3,30 @@
...
+	ORGANIZATION -> PROJECT
+	ORGANIZATION -> ORGANIZATION
```
Comment (tsmaeder) [inline _blueprints/model_overview.adoc:17]: Nested orgs?

Comment (aslakknutsen) [reply _blueprints/model_overview.adoc:17]: Yes, according to the latest from PM we have that....

File: _blueprints/model_overview.adoc (hunk 1 above)
Comment (aslakknutsen) [reply _blueprints/model_overview.adoc:15]: In this model an Org controls the Teams and a Team has access to a Project.

Maybe there are two layers of Teams; Org Wide and Project local ?

Comment (tsmaeder) [reply _blueprints/model_overview.adoc:15]: As I hinted in my mail re: projects, I think teams (groups) have a life independent of projects.

**Review by tsmaeder — APPROVED** [review]

File: _blueprints/model_overview.adoc (hunk 1 above)
Comment (aslakknutsen) [reply _blueprints/model_overview.adoc:15]: Ah, ic what you mean. Replace TEAM->PROJECT with PERMISSION->PROJECT

File: _blueprints/model_overview.adoc (hunk 2 above)
Comment (maxandersen) [reply _blueprints/model_overview.adoc:17]: Did I miss something - where did that show up ? thats going to make things really complicated.

Comment (aslakknutsen) [reply _blueprints/model_overview.adoc:17]: @maxandersen Look at the drawing from Pete that you've been commenting on.. :)

**Pull request merged**